res/themes/dark/css/_dark.scss

```
@@ -72,7 +72,7 @@ $avatar-bg-color: $bg-color;
 
 $h3-color: $primary-fg-color;
 
-$dialog-title-fg-color: #454545;
+$dialog-title-fg-color: #edf3ff;
```

For the dark theme, I tried to make values flow from the top "unified palette" section if there was a match.

Maybe:

```suggestion
$dialog-title-fg-color: $base-text-color;
```